plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/PlatformUtil.java

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************* 
+ * Copyright (c) 2015 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.eclipse.swt.browser;
+
+/**
+ * @author Ilya Buziuk (ibuziuk)
+ */
+public final class PlatformUtil {
```

Comment:
I have already seen a few utils with for similar purposes. Can't we reuse some already existing one or create it somewhere where we can reuse it across JBT?